### PR TITLE
Implement logic to display order data in admin

### PR DESCRIPTION
**Pull request by ArjenMiedema**
On the order view page in the admin it now shows the order data fetched from Exact, like attachments, order item information and order information.

### CODE REVIEW

File: src/Block/Adminhtml/Order/View/Attachments.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+/**
+ * Copyright Youwe. All rights reserved.
+ * https://www.youweagency.com
+ */
+
+declare(strict_types=1);
+
+namespace JcElectronics\ExactOrders\Block\Adminhtml\Order\View;
+
+class Attachments extends \Magento\Sales\Block\Adminhtml\Order\AbstractOrder
```
Comment (ar-vie) [inline src/Block/Adminhtml/Order/View/Attachments.php:12]: \Magento\Sales\Block\Adminhtml\Order\AbstractOrder could be imported via `use`

File: src/ViewModel/Adminhtml/Order/View/Customer.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * Copyright Youwe. All rights reserved.
+ * https://www.youweagency.com
+ */
```
Comment (ar-vie) [inline src/ViewModel/Adminhtml/Order/View/Customer.php:6]: /**

Copyright JC-Electronics. All rights reserved.
https://www.jc-electronics.com/
*/

File: src/etc/adminhtml/di.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Youwe. All rights reserved.
+ * https://www.youweagency.com
+ */
+-->
```
Comment (ar-vie) [inline src/etc/adminhtml/di.xml:7]: /**

Copyright JC-Electronics. All rights reserved.
https://www.jc-electronics.com/
*/

File: src/view/adminhtml/templates/order/view/attachments.phtml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+<?php
+
```
Comment (ar-vie) [inline src/view/adminhtml/templates/order/view/attachments.phtml:2]: copyrights comment can be added

File: src/view/adminhtml/templates/order/view/exact_customer_info.phtml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+/**
+ * @var Template $block
+ * @var Escaper $escaper
+ */
+
+use JcElectronics\ExactOrders\ViewModel\Adminhtml\Order\View\Customer;
+use Magento\Framework\Escaper;
+use Magento\Framework\View\Element\Template;
+
+/** @var \JcElectronics\ExactOrders\ViewModel\Adminhtml\Order\View\Customer $customerViewModel */
```
Comment (ar-vie) [inline src/view/adminhtml/templates/order/view/exact_customer_info.phtml:12]: Could be shorted to 
/** @var  Customer $customerViewModel */

**Review by ar-vie — APPROVED** [review]

**Review by CyberSecutor — APPROVED** [review]

**Pull request merged**